packages/cli/src/tokens/write.ts

```diff
@@ -1,18 +1,23 @@
 import fs from 'node:fs/promises';
 import path from 'node:path';
+import type { ThemeObject } from '@tokens-studio/types';
+import chalk from 'chalk';
 import * as R from 'ramda';
 import type { ColorMode } from '../colors/types.js';
 import type { Collection, File, Tokens, TokensSet, TypographyModes } from './types.js';
 import { generateMetadataJson } from './write/generate$metadata.js';
 import { generateThemesJson } from './write/generate$themes.js';
 
 const DIRNAME: string = import.meta.dirname || __dirname;
-const DEFAULT_FILES_PATH = path.join(DIRNAME, '../../init/template/default-files/design-tokens/');
+const DEFAULT_FILES_PATH = path.join(DIRNAME, './design-tokens/default');
+const TEMPLATE_FILES_PATH = path.join(DIRNAME, './design-tokens/template/');
+
+export const stringify = (data: unknown) => JSON.stringify(data, null, 2).replace(/\r\n/g, '');
```

Comment:
`\r\n` (crlf) smells like missing line ending normalization in the repository, but `.gitattributes` has
```
* text=auto eol=lf
```
So does this actually do anything?

Reply:
I had to remove the line ending from the design-tokens files because Token Studio strips them once imported, so a lot of files was flagged as changed the first time you use them.

Reply:
That makes sense, but I would think it should be `.replace(/\n/g, '')`. On my system, there are no files in the repo which have `\r\n` in them, and `eol=lf` in `.gitattributes` SHOULD force Git to normalize line endings to `\n` on all systems on checkout, so I don't understand how this actually does any replacement of line endings.

Alternatively `/\r?\n/g` should work regardless of system (only Windows uses `\r\n` for line endings by default) but it's weird if you have `\r\n` in any files now with the current `.gitattributes` file

Reply:
I tried `.replace(/\n/g, '')`first time but it removes all the new lines after each prop so file ended up just gibberish. Same with `/\r?\n/g`. Maybe its due to the spacer parameter to `JSON.stringify`. 

<img width="1285" alt="image" src="https://github.com/user-attachments/assets/8425eaf0-bfa8-4d7b-bf6f-da820ef35af3">


I did some trial and error before landing on the one used now that works. 🤷

Reply:
Well, on my end the script works the same way with and without `.replace(/\r\n/g, '');` so I don't think that particular invocation does anything, unless you have added a Windows line feed to your file in an editor which doesn't detect existing line endings, and not commited it

Reply:
To be clear I think the line should just be
```
export const stringify = (data: unknown) => JSON.stringify(data, null, 2)
```

 since this already strips the newline before the end of the file

Reply:
ahh, true. Same here. I'll just remove it then and we can add it later if its needed again.